src/power_process.c

```diff
@@ -381,7 +383,8 @@ void draw_god_lightning(struct Thing *shotng)
         locpos.x.val = (shotng->mappos.x.val + (LbSinL(i + cam->orient_a) >> (LbFPMath_TrigmBits - 10))) + 128;
         locpos.y.val = (shotng->mappos.y.val - (LbCosL(i + cam->orient_a) >> (LbFPMath_TrigmBits - 10))) + 128;
         locpos.z.val = shotng->mappos.z.val + subtile_coord(12,0);
-        draw_lightning(&locpos, &shotng->mappos, 256, TngEffElm_ElectricBall3);
+        struct ShotConfigStats* shotst = get_shot_model_stats(ShM_GodLightBall);
+        draw_lightning(&locpos, &shotng->mappos, 256, shotst->effect_id);
```

Comment:
the eeinterspace for lightning should probably also be made configurable, it is the difference between drain and lightning (and the color of course).